src/test/java/com/google/api/codegen/testdata/library_config.baseline

```diff
@@ -283,3 +283,14 @@ interfaces:
     field_name_patterns:
       name: book_2
     timeout_millis: 30000
+  - name: StreamShelves
+    request_object_method: false
+    page_streaming:
```

Comment:
@vchudnov-g Somewhat interestingly, gRPC streaming and page streaming are not mutually exclusive. For example, an API might accept a page token and return a stream of (page token and an array of objects). We might want to "flatten" the stream. The behavior might depend on language, but the Go team has asked for this behavior.